service/api.go

```diff
@@ -155,6 +156,22 @@ func (s *gnomobileService) GetActiveAccount(ctx context.Context, req *connect.Re
 	return connect.NewResponse(&rpc.GetActiveAccountResponse{Key: info}), nil
 }
 
+// DeleteAccount deletes the account with the given name, using the password to
+// ensure access. If the account doesn't exist, then return ErrCryptoKeyNotFound.
+// If the password is wrong, then return ErrDecryptionFailed.
+func (s *gnomobileService) DeleteAccount(ctx context.Context, req *connect.Request[rpc.DeleteAccountRequest]) (*connect.Response[rpc.DeleteAccountResponse], error) {
+	if err := s.getSigner().Keybase.Delete(req.Msg.NameOrBech32, req.Msg.Password, false); err != nil {
+		if strings.Contains(err.Error(), "not found") {
+			return nil, rpc.ErrCode_ErrCryptoKeyNotFound
+		} else if strings.Contains(err.Error(), "ciphertext decryption failed") {
```

Comment:
It's better to compare against a const error message exported from Gno. Is it possible?

Reply:
OK. I pushed a comment to check the error type.
https://github.com/gnolang/gnomobile/pull/43/commits/c309af58475da3aefb13b1c0a81aad1bb34872b1